.deepsource.toml

```diff
@@ -14,6 +14,11 @@ exclude_patterns = [
 [[analyzers]]
 name = "python"
 enabled = true
+dependency_file_paths = [
+    "requirements/requirements.txt",
+    "requirements/requirements-nightly.txt",
+    "requirements/requirements-dev.txt"
+]
```

Comment:
Would this work?

```suggestion
dependency_file_paths = [
    "requirements/*"
]
```

Reply:
No, `requirements/README.md` is not a requirements file and `requirements/pre-requirements.txt` does not actually contain any useful dependency information.